apps/web-giddh/src/assets/locale/invoice/hi.json

```diff
@@ -142,6 +142,15 @@
 		"na": "इस लेनदेन के लिए ई-चालान निर्माण लागू नहीं है।",
         "giddh_invoice_not_cancelled": "ई-चालान लेनदेन आईआरपी में रद्द कर दिया गया है लेकिन गिद्ध में नहीं। इससे जुड़ा IRN अब मान्य नहीं है।"
     },
+    "e_invoice_statuses_label": {
```

Comment:
These status will come from GST portal so will be in English for all the language